### PR TITLE
Add documentation in regards to issues/pull request templates

### LINKED ISSUE #3491: Issue and PR templates
- Gitea version (or commit ref): 1.4.0+rc1
- Git version: 2.16.1
- Operating system: Linux (Arch Linux)
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:
*Not relevant*

## Description
I have absolutely no idea how I can word this, but I'll do my best.

In a nutshell, if the template file exists in the repository (let's say `.gitea/ISSUE_TEMPLATE.md`), then Gitea can fetch the contents of the file and populate the textarea on the new issue template. Otherwise leave the textarea empty.

For example, GitHub implements this feature.

*PS: my wording can be terrible; English isn't my native language*

### PR DESCRIPTION
**Pull request by techknowlogick**
Fixes #3491

### CODE REVIEW

File: docs/content/doc/usage/issue-pull-request-templates.en-us.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+For some projects there are a standard list of questions that users need to be asked
+for creating an issue, or adding a pull request. Gitea supports adding templates to the
+main branch of the repository so that they can autopopulate the form when users are 
+creating issues, and pull requests. This will cut down on the initial back and forth
+of getting some clarifiying details.
+
+Possible file names for issue templates:
+
+```
+"ISSUE_TEMPLATE.md",
```
Comment (kolaente) [inline docs/content/doc/usage/issue-pull-request-templates.en-us.md]: What about making this a bullet-point list instead (aka deleting `"` and `,`)?

Comment (techknowlogick) [reply docs/content/doc/usage/issue-pull-request-templates.en-us.md]: thanks @kolaente I've updated the PR to match your suggestion.

**Review by appleboy — APPROVED** [review]

**Review by kolaente — APPROVED** [review]

**Comment by techknowlogick** [comment]
@kolaente sadly you'll need to be a maintainer to approve PRs. As we've talked about before I think you have enough accepted PRs to be a maintainer, if you apply.

**Comment by thehowl** [comment]
LGTM

**Review by jonasfranz — APPROVED** [review]

**Pull request merged**